client/src/components/SideBanner/SideBanner.js

```diff
@@ -30,6 +30,11 @@ const useStyles = makeStyles({
 const SideBanner = (props) => {
   const classes = useStyles();
   const { children } = props;
+
+  const smallScreen = useMediaQuery(theme => theme.breakpoints.down('sm'));
+  if (smallScreen) {
+    return children;
+  }
```

Comment:
You might be able to use [Hidden](https://material-ui.com/components/hidden/#hidden) in the parent to remove the side banner on small screens